scripts/run_latest_build.sh

```diff
@@ -72,14 +72,18 @@ DOCKERHUB_ORG=${DOCKERHUB_ORG:-"ansibleplaybookbundle"} # DocherHub org where AP
 ENABLE_BASIC_AUTH=${ENABLE_BASIC_AUTH:-"false"} # Secure broker with basic authentication, default 'false'. Disabling basic auth allows "apb push" to work.
 BROKER_KIND=${BROKER_KIND:-"Broker"} # allow users to override the broker kind type to work with 3.7
 
+auth=$(echo -e "{\"basicAuthSecret\":{\"namespace\":\"ansible-service-broker\",\"name\":\"asb-auth-secret\"}}")
+BROKER_AUTH=${BROKER_AUTH:-"${auth}"}
+
 curl -s $TEMPLATE_URL \
   | oc process \
   -n ansible-service-broker \
   -p DOCKERHUB_USER="$DOCKERHUB_USER" \
   -p DOCKERHUB_PASS="$DOCKERHUB_PASS" \
   -p DOCKERHUB_ORG="$DOCKERHUB_ORG" \
   -p BROKER_KIND="$BROKER_KIND" \
-  -p ENABLE_BASIC_AUTH="$ENABLE_BASIC_AUTH" -f - | oc create -f -
+  -p ENABLE_BASIC_AUTH="$ENABLE_BASIC_AUTH" -f - | oc create -f - \
+  -p BROKER_AUTH="$BROKER_AUTH"
```

Comment:
Need to move line 86 above 85. BROKER_AUTH isn't being processed with the template command.